omnibus/config/software/jmxfetch.rb

```diff
@@ -26,5 +25,8 @@
   ship_license "https://raw.githubusercontent.com/DataDog/jmxfetch/master/LICENSE"
   mkdir jar_dir
   copy "jmxfetch-#{jmxfetch_version}-jar-with-dependencies.jar", jar_dir
-  block { File.chmod(0644, "#{jar_dir}/jmxfetch-#{jmxfetch_version}-jar-with-dependencies.jar") }
+  block do
+    File.rename("#{jar_dir}/jmxfetch-#{jmxfetch_version}-jar-with-dependencies.jar", "#{jar_dir}/jmxfetch.jar")
```

Comment:
nit: we can probably achieve the same result by changing the target path of the `copy` above.

Reply:
Indeed, it works with the copy.